packages/lesswrong/components/messaging/ModeratorInboxWrapper.tsx

```diff
@@ -4,21 +4,37 @@ import { useLocation } from '../../lib/routeUtil';
 import { useCurrentUser } from '../common/withUser';
 import { userCanDo } from '../../lib/vulcan-users';
 import { Link } from '../../lib/reactRouterWrapper';
+import { isEAForum } from '../../lib/instanceSettings';
 
 const ModeratorInboxWrapper = () => {
-  const { ErrorAccessDenied } = Components
   const currentUser = useCurrentUser();
-  const { query } = useLocation();
-  
+  const { query, params } = useLocation();
+
+  const { InboxNavigation, FriendlyInbox } = Components
+
   if (!currentUser) {
     return <div>Log in to access private messages.</div>
   }
-  if (!userCanDo(currentUser, 'conversations.view.all')) {
-    return <ErrorAccessDenied/>
-  }
+
+  const conversationId = params._id;
+
   const showArchive = query.showArchive === "true"
-  const terms: ConversationsViewTerms = {view: 'moderatorConversations', showArchive, userId: query.userId};
-  return <Components.InboxNavigation terms={terms} currentUser={currentUser} title={<Link to="/moderatorInbox">Moderator Conversations</Link>}/>
+  const terms: ConversationsViewTerms = { view: "moderatorConversations", showArchive, userId: query.userId };
```

Comment:
This component is essentially the same as `InboxWrapper` but with different `terms`